modules/nw-egressnetworkpolicy-about.adoc

```diff
@@ -17,6 +17,11 @@ endif::[]
 [id="nw-egressnetworkpolicy-about_{context}"]
 = How an egress firewall works in a project
 
+[NOTE]
+====
+Network policy does not apply to the host network namespace. Pods with host networking enabled are unaffected by network policy rules.
```

Comment:
Please update "Network policy", "network policy " to "egress firewall", they are two different concepts.

Reply:
acked